docs/architecture/porting-existing-aspnet-apps/deployment-scenarios.md

```diff
@@ -9,15 +9,65 @@ ms.date: 11/13/2020
+
+Configured in this manner, the only change that must be made in order for the ASP.NET Core app's APIs to be routed properly is to change its default route template from `[Route("[api/controller]")]` to `[Route("[controller]")]`.
+
+Alternately the ASP.NET Core app can be another top level web site in IIS. In this case, you can configure the original site to use a rewrite rule (with [URL Rewrite](https://www.iis.net/downloads/microsoft/url-rewrite)) that will redirect to the other app if the path starts with `/api`. The ASP.NET Core app can use a different host header for its route so that it doesn't conflict with the main app but can still respond to requests using root-based routes.
+
+As an example, the same ASP.NET Core app used in Figure 5-1 can be deployed to another folder configured as an IIS web site. The site should use an application pool configured just as before, with `No Managed Code`. Configure its bindings to respond to a unique host name on the server, such as `api.contoso.com`. To configure URL Rewrite to rewrite requests matching `/api` simply add a new inbound rule at the IIS server (or individual site) level. Match the pattern `^/api(.*)` and specify an Action type of `Rewrite` and a Rewrite URL of `api.contoso.com/{R:1}`. The combination of using `(.*)` in the matching pattern and `{R:1}` in the rewrite URL will ensure the rest of the path gets used with the new URL. With this in place, separate sites on the same IIS server can coexist running separate versions of .NET, but they can be made to appear to the Internet as one web application. Figure 5-2 shows the rewrite rule as configured in IIS with the separate web site.
+
+![IIS Manager showing URL Rewrite rule to route subfolder requests to separate web site](./media/Figure5-2.png)
+
+**Figure 5-2**. Rewrite rule to rewrite subfolder requests to another web site.
+
+If your app requires single sign-on between different sites or applications within IIS, refer to the documentation on [how to share authentication cookies among ASP.NET apps](https://docs.microsoft.com/aspnet/core/host-and-deploy/iis/) for detailed instructions on supporting this scenario.
```

Comment:
```suggestion
If your app requires a single sign-on between different sites or applications within IIS, refer to the documentation on [how to share authentication cookies among ASP.NET apps](https://docs.microsoft.com/aspnet/core/host-and-deploy/iis/) for detailed instructions on supporting this scenario.
```

Reply:
"single sign-on" or SSO is a feature of an app or collection of apps. I think if it were using the acronym it would sound better to say "your app uses SSO" than "your app uses an SSO" so I wouldn't add an article in front of it here.

Reply:
I agree with Steve on this one, I do not believe the suggestion is correct to add "a"